job-dsls/jobs/pr_jobs.groovy

```diff
@@ -206,12 +206,20 @@ for (repoConfig in REPO_CONFIGS) {
                 cron("")
                 triggerPhrase(".*[j|J]enkins,?.*(retest|test) this.*")
                 allowMembersOfWhitelistedOrgsAsAdmin(true)
-                whiteListTargetBranches {
-                    ghprbBranch {
-                        branch("${repoBranch}")
+                if ( repo == "optaplanner" ) {
+                    whiteListTargetBranches {
+                        ghprbBranch {
+                            branch("7.x")
+                        }
                     }
-                    ghprbBranch {
-                        branch("7.x")
+                } else {
+                    whiteListTargetBranches {
+                        ghprbBranch {
+                            branch("${repoBranch}")
+                        }
+                        ghprbBranch {
+                            branch("7.x")
+                        }
```

Comment:
I would suggest to replace your changes by
```
whiteListTargetBranches {
    if ( repo != "optaplanner" ) {
        ghprbBranch {
            branch("${repoBranch}")
        }
    }
    ghprbBranch {
        branch("7.x")
    }
}
```
wdyt @mbiarnes ?

Reply:
@Ginxo are we not interested in having master and 7.x branch in optaweb-employee-rostering and optaweb-vehicle-routing?

Reply:
@mbiarnes I can't tell about business here, just saying your PR proposal
```
if ( repo == "optaplanner" ) {
    whiteListTargetBranches {
        ghprbBranch {
            branch("7.x")
        }
    }
} else {
    whiteListTargetBranches {
        ghprbBranch {
            branch("${repoBranch}")
        }
        ghprbBranch {
            branch("7.x")
        }
    }
}
```
can be replace by this which is simpler 
```
whiteListTargetBranches {
    if ( repo != "optaplanner" ) {
        ghprbBranch {
            branch("${repoBranch}")
        }
    }
    ghprbBranch {
        branch("7.x")
    }
}
```

Reply:
DONE

Reply:
@Ginxo mrge when you're OK with it